prow/plugins/verify-owners/verify-owners.go

```diff
@@ -43,6 +43,7 @@ const (
 	PluginName                    = "verify-owners"
 	ownersFileName                = "OWNERS"
 	ownersAliasesFileName         = "OWNERS_ALIASES"
+	vendorDir                     = "vendor"
```

Comment:
Can we make this more generic?

This seems awfully hard-coded to old versions of golang

Reply:
I don't think we are tying to solve a generic problem

Reply:
Current solution is too short sighted
https://github.com/kubernetes/kubernetes/tree/master/third_party

Reply:
I'm not thrilled to expose this as config since this is going to be one of those "you must set this option for reasonable function" sort of options -- are we OK to just have a set of blacklisted directories?

Reply:
I'm fine with there being a non-empty default value so long as we can change it.

The vendor dir is overfit for kubernetes and golang.